src/test/unit/System.Windows.Forms/System/Windows/Forms/ButtonRendererTests.cs

```diff
@@ -0,0 +1,281 @@
+    [InlineData(PushButtonState.Default, "Button", TextFormatFlags.SingleLine, false)]
+    public void DrawButton_TextFontFlagsImageImageBoundsFocused_DoesNotThrow(
+    PushButtonState state,
+    string? buttonText,
+    TextFormatFlags flags,
+    bool focused)
+    {
+        using Bitmap bitmap = new(160, 70);
+        using Graphics graphics = Graphics.FromImage(bitmap);
+        Rectangle bounds = new(0, 0, 160, 70);
+        using Font font = new("Verdana", 13);
+        using Bitmap image = new(32, 32);
+        Rectangle imageBounds = new(30, 20, 32, 32);
+
+        Exception? exception = Record.Exception(() =>
+            ButtonRenderer.DrawButton(graphics, bounds, buttonText, font, flags, image, imageBounds, focused, state));
+        exception.Should().BeNull();
+    }
+
+    [WinFormsTheory]
+    [InlineData(PushButtonState.Normal, "Advanced", TextFormatFlags.Default, true)]
+    [InlineData(PushButtonState.Normal, "Advanced", TextFormatFlags.Default, false)]
+    [InlineData(PushButtonState.Normal, null, TextFormatFlags.HorizontalCenter | TextFormatFlags.VerticalCenter, true)]
+    [InlineData(PushButtonState.Normal, "Button", TextFormatFlags.SingleLine, false)]
+    [InlineData(PushButtonState.Hot, "Advanced", TextFormatFlags.Default, true)]
+    [InlineData(PushButtonState.Hot, "Advanced", TextFormatFlags.Default, false)]
+    [InlineData(PushButtonState.Hot, null, TextFormatFlags.HorizontalCenter | TextFormatFlags.VerticalCenter, true)]
+    [InlineData(PushButtonState.Hot, "Button", TextFormatFlags.SingleLine, false)]
+    [InlineData(PushButtonState.Pressed, "Advanced", TextFormatFlags.Default, true)]
+    [InlineData(PushButtonState.Pressed, "Advanced", TextFormatFlags.Default, false)]
+    [InlineData(PushButtonState.Pressed, null, TextFormatFlags.HorizontalCenter | TextFormatFlags.VerticalCenter, true)]
+    [InlineData(PushButtonState.Pressed, "Button", TextFormatFlags.SingleLine, false)]
+    [InlineData(PushButtonState.Disabled, "Advanced", TextFormatFlags.Default, true)]
+    [InlineData(PushButtonState.Disabled, "Advanced", TextFormatFlags.Default, false)]
+    [InlineData(PushButtonState.Disabled, null, TextFormatFlags.HorizontalCenter | TextFormatFlags.VerticalCenter, true)]
+    [InlineData(PushButtonState.Disabled, "Button", TextFormatFlags.SingleLine, false)]
+    [InlineData(PushButtonState.Default, "Advanced", TextFormatFlags.Default, true)]
+    [InlineData(PushButtonState.Default, "Advanced", TextFormatFlags.Default, false)]
+    [InlineData(PushButtonState.Default, null, TextFormatFlags.HorizontalCenter | TextFormatFlags.VerticalCenter, true)]
+    [InlineData(PushButtonState.Default, "Button", TextFormatFlags.SingleLine, false)]
+    public void DrawButton_IDeviceContext_TextFontFlagsImageImageBoundsFocused_DoesNotThrow(
+    PushButtonState state,
+    string? buttonText,
+    TextFormatFlags flags,
+    bool focused)
```

Comment:
add indents here and other places as well
```suggestion
    public void DrawButton_IDeviceContext_TextFontFlagsImageImageBoundsFocused_DoesNotThrow(
        PushButtonState state,
        string? buttonText,
        TextFormatFlags flags,
        bool focused)
```